lib/src/model/package_graph.dart

```diff
@@ -230,8 +203,8 @@ class PackageGraph with CommentReferable, Nameable {
   // more than once for them.
   final Map<Element, ModelNode> _modelNodes = {};
 
-  /// The collection of "special" classes for which we need some special access.
-  final specialClasses = SpecialClasses();
+  /// The Object class declared in the Dart SDK's 'dart:core' library.
+  late InheritingContainer objectClass;
```

Comment:
Looks like this broke dartdoc analysis for certain packages (looks like when mixins are used). See #3947